app.js

```diff
@@ -101,12 +115,12 @@ app.get('/rideInfo/:rideId', function(req, res){
   mongoose.model('rideInfo').find({ 'rideId': req.params.rideId }, function (err, result) {
     if (err) {
       res.send(404, { 'success': false, error: { 'message': err } });
-      console.log('ERRORED at GET /   , ');
-      return console.error(err);
+      log.error('ERRORED at GET /   , ' + err);
+      return ;
     }
     else {
-      console.log('SUCCESS at GET /   , for collection: rideInfo');
-      console.log('Get results for RIDE ID =  ' + req.params.rideId );
+      log.info('SUCCESS at GET /   , for collection: rideInfo');
+      log.info('Get results for RIDE ID =  ' + req.params.rideId );
```

Comment:
Combine these two messages in one... something like `'Successfully got results for rideId = ' + rideId`.